VERSION

```diff
@@ -41,7 +41,7 @@ flex_min_version=2.5.4
 # requirement is that it must be entirely printable ASCII characters
 # and have no white space.
 
-greek=rc2
+greek=
```

Comment:
We don't commit with greek equal to nothing; doesn't the release jenkins job take care of building with greek=nothing when making the release tarball?

Reply:
man, you are right - and also - I've been doing it wrong this entire time.... *sigh*

Reply:
@jsquyres done - can please re-review/approve